index.js

```diff
@@ -43,6 +43,11 @@ const filesToCopyFromRootToAddonInAddonOnlyMode = [
   'README.md',
 ];
 
+const dependenciesToKeepInSync = [
```

Comment:
This is a good thing to do for monorepos -- but I think we should probably match versions for _all_ dependencies that match both package.jsons, rather than just these two.

Right now, it's just an unfortunate circumstance that the eslint ecosystem pre-flat-config assumes everything will be hoisted to the top-level node_modules.